apis/python/setup.py

```diff
@@ -111,7 +111,7 @@ def run(self):
         "scanpy",
         "scipy",
         "somacore==0.0.0a7",
-        "tiledb>=0.20.0",
+        "tiledb>=0.20.0,<0.21.0",
```

Comment:
You can use the [`~=` operator or `*` matcher](https://peps.python.org/pep-0440/#compatible-release) here:

`tiledb~=0.20.0` or `tiledb==0.20.*`